core/primitives/res/runtime_configs/49.txt

```diff
@@ -0,0 +1,2 @@
+wasm_regular_op_cost: 822756
```

Comment:
I wonder if it makes to spell this as

```
wasm_regular_op_cost: 822756 -> 2207874
```

Extra check, and somewhat better readability. 

Not sure if its worth the extra impl complexity

Reply:
I like it a lot! In the current format, implementing this would be simple enough, I think. But I will wait to see if we might want to go to a different format instead.